index.js

```diff
@@ -83,9 +87,15 @@
         }, $this.data('load-after'));
       });
     } else {
-      whenScroll(['within 300px of', $newElement[0]], function () {
-        loadNow($this, $newElement);
-      }, true);
+      // This is essential for firing the event to a DOM immediately, as set by
```

Comment:
(I'm going to assume you're not winding me up)

Yep: https://help.github.com/articles/fork-a-repo/